error message in server log

Upon startup, I see this in the server log
```
[2020-11-27 06:39:15.604] [ERROR] console - UNSETTLED FUNCTION BUG IN HOOK
FUNCTION (plugin: ep_pad-lister, function name:
/opt/etherpad-lite/node_modules/ep_pad-lister/index:registerRoute, hook:
expressCreateServer): The hook function neither called the callback nor
returned a non-undefined value. This is prohibited because it will result in
freezes when a future version of Etherpad updates the hook to support
asynchronous behavior.
```

ep_pad-lister 1.2.1
ep 1.8.6-248-g750c7cb

--- a/index.js
+++ b/index.js
@@ -15,7 +15,7 @@ exports.eejsBlock_indexWrapper = function (hook_name, args, cb) {
   return cb();
 };
 
-exports.registerRoute = function (hook_name, args, cb) {
+exports.registerRoute = function (hook_name, args) {
   args.app.get('/pad-lister/static/bootstrap.min.css', (req, res) => {
     res.sendFile(__dirname + '/static/css/bootstrap.min.css');
   });

